admin/billing/payment/payment.go

```diff
@@ -16,6 +18,12 @@ type Provider interface {
 	DeleteCustomer(ctx context.Context, customerID string) error
 	// GetBillingPortalURL returns the payment portal URL to collect payment information from the customer.
 	GetBillingPortalURL(ctx context.Context, customerID, returnURL string) (string, error)
+
+	// WebhookHandlerFunc returns a http.HandlerFunc that can be used to handle incoming webhooks from the payment provider. Return nil if you don't want to register any webhook handlers.
+	WebhookHandlerFunc(ctx context.Context) func(w http.ResponseWriter, r *http.Request)
+
+	// SetJobsClient needs to be explicitly set because of circular dependency, see admin start method
+	SetJobsClient(jobs jobs.Client)
```

Comment:
The same comments as on `billing.Client` about `http.HandleFunc` and passing `jobs.Client` to the handler apply here